components/ResultsPage/YourAnswers.tsx

```diff
@@ -37,7 +37,7 @@ export const YourAnswers: React.VFC<{
               <strong>{getDisplayValue(input)}</strong> &nbsp;
               <DSLink
                 id={`edit-${fieldKey}`}
-                href={`/eligibility#${fieldKey}`}
+                href={`${tsln.eligibility}#${fieldKey}`}
```

Comment:
On French mode, the link could not navigate to the accordion form, instead, it shows "we could not find the web page"